src/test/server/server_test_runner.cpp

```diff
@@ -40,6 +41,8 @@ class ServerTestRunner : public BaseTest {
     // Give the server time to shut down gracefully before force-closing the socket it's working on
     std::this_thread::sleep_for(std::chrono::milliseconds(10));
     _server_thread->join();
+
+    std::remove(_export_filename.c_str());
```

Comment:
What happens to following tests if the process dies before this is reached?

Reply:
A following test here (either `TestCopyExport` or `TestCopyIntegration`) could accidentally be green as we do not compare the files. I could copy the file comparison from `BinaryWriterTest` to at least make sure the other test would fail in advance. For re-running, if the file still exists, we could still have a false positive, which is also the case for any other export test. Therefore, I think removing the file in `SetUp()` rather than in `TearDown()` would be a better  solution.